.gitlab/subscribed-pipelines.yml

```diff
@@ -21,6 +21,7 @@
       if [[ $(jq '.[env.CI_MACHINE].total_nodes_up' /usr/global/tools/lorenz/data/loginnodeStatus) == 0 ]]
       then
         echo -e "\e[31mNo node available on ${CI_MACHINE}\e[0m"
+        false && \
         curl --url "https://api.github.com/repos/${GITHUB_PROJECT_ORG}/${GITHUB_PROJECT_NAME}/statuses/${CI_COMMIT_SHA}" \
```

Comment:
I'm not sure what's the idea of this `curl` command, and if we want it, so I disabled it.

Reply:
This is how you will know, directly in GitHub, that the pipeline failed because the machine wasn't available, instead of having to click the link, login to gitlab, and realize that the jobs did not run.

Reply:
I don't think we want to post on GitHub if a machine was down during a Gitlab CI run.